pkg/pods/pod_test.go

```diff
@@ -203,41 +203,60 @@ func TestBuildRunitServices(t *testing.T) {
 		path:           "/data/pods/testPod",
 		ServiceBuilder: serviceBuilder,
 		Chpst:          hoist.FakeChpst(),
+		Contain:        hoist.FakeContain(),
 	}
 	hoistLaunchable := hoist.FakeHoistLaunchableForDir("multiple_script_test_hoist_launchable")
 	hoistLaunchable.RunAs = "testPod"
+	hoistLaunchable.Contain = hoist.FakeContain()
+	hoistLaunchable.ContainerType = "mycgroup"
 	executables, err := hoistLaunchable.Executables(serviceBuilder)
 	outFilePath := path.Join(serviceBuilder.ConfigRoot, "testPod.yaml")
 
 	Assert(t).IsNil(err, "Got an unexpected error when attempting to start runit services")
 
 	pod.BuildRunitServices([]hoist.HoistLaunchable{*hoistLaunchable})
-	expected := fmt.Sprintf(`%s:
+	expected := fmt.Sprintf(`%[1]s:
   run:
   - /usr/bin/nolimit
-  - %s
+  - %[6]s
```

Comment:
I don't think it's that bad. template is more powerful but the API is also much less convenient to use. Explicit indices are enough to avoid duplicating arguments, and it's obvious that the indices are significant so we can avoid tripping over them in the future.